### PR TITLE
rolling-router-sticky-sessions ui added

### PR DESCRIPTION
**Pull request by mikarinneoracle**

### CODE REVIEW

File: ContainerCloud/images/rolling-router-sticky-sessions/Dockerfile.template [hunk 1]
```diff
@@ -3,11 +3,14 @@ FROM __REGISTRY_NAME__/confd:__VERSION_CONFD__
...
     build_pkgs="build-base linux-headers openssl-dev pcre-dev wget zlib-dev" && \
     runtime_pkgs="ca-certificates openssl pcre zlib" && \
-    apk --update add ${build_pkgs} ${runtime_pkgs} && \
+    php_pkgs="php5-fpm php5-mcrypt php5-soap php5-openssl php5-gmp php5-pdo_odbc php5-json php5-dom php5-pdo php5-zip php5-mysql php5-mysqli php5-sqlite3 php5-apcu php5-pdo_pgsql php5-bcmath php5-gd php5-xcache php5-odbc php5-pdo_mysql php5-pdo_sqlite php5-gettext php5-xmlreader php5-xmlrpc php5-bz2 php5-memcache php5-mssql php5-iconv php5-pdo_dblib php5-curl php5-ctype" && \
+    apk --update add ${build_pkgs} ${runtime_pkgs} ${php_pkgs} && \
     cd /tmp && \
     mkdir unzip && \
     wget https://bitbucket.org/nginx-goodies/nginx-sticky-module-ng/get/08a395c66e42.zip && \
     unzip 08a395c66e42.zip -d unzip && \
+    ls -la /tmp/unzip && \
```
Comment (karthequian) [inline ContainerCloud/images/rolling-router-sticky-sessions/Dockerfile.template]: @mikarinneoracle why do we need this?

Comment (mikarinneoracle) [inline ContainerCloud/images/rolling-router-sticky-sessions/Dockerfile.template]: We don't. Can be removed.

Comment (mikarinneoracle) [inline ContainerCloud/images/rolling-router-sticky-sessions/Dockerfile.template]: Removed.

**Comment by mikarinneoracle** [comment]
@karthequian Should I fix the code indentation? I realized my Atom is using tabs, should I change to spaces (4) instead ?

**Comment by karthequian** [comment]
@mikarinneoracle I did not see that initially, but I agree. 4 spaces would work better, especially in the php/js code you have. It might make reading it easier, and more consistent with the rest of the files. Thanks for the PR!

**Comment by mikarinneoracle** [comment]
@karthequian Ok, indentation fixed for readability. Thanks!

**Comment by mikarinneoracle** [comment]
@karthequian  @mikeraab Hi, I updated the README and did a small fix to error handling in case of network connect error. Please review.

**Comment by karthequian** [comment]
looks good to me.

**Comment by mikeraab** [comment]
Thanks Guys, merging now.

**Pull request merged**